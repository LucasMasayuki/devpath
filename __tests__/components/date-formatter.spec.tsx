import React from 'react'
import renderer from 'react-test-renderer';
import DateFormatter from '@/components/date-formatter';

describe('DateFormatter', () => {
  it('should render formatted time', () => {
    const date = '1999-03-01'
    const component = renderer.create(
      <DateFormatter dateString={date} />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})