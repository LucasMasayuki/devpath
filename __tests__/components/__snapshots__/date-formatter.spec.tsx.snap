// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateFormatter should render formatted time 1`] = `
<time
  dateTime="1999-03-01"
>
  March	1, 1999
</time>
`;
